release/requirements_buildkite.in

```diff
@@ -8,7 +9,7 @@ freezegun
 google-cloud-storage
 jinja2
 protobuf >= 3.15.3, != 3.19.5
-pydantic < 1.10.0
+pydantic >= 2.5.0
```

Comment:
or just remove pydantic i think it's no longer a requirement

Reply:
Should I also remove `anyscale`?